modules/nf-core/agat/spaddintrons/environment.yml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
```

Comment:
```suggestion
```

Reply:
thanks, i thought this was necessary markup for validating the file.